domains/mraxays.json

```
@@ -5,6 +5,11 @@
   },
   "records": {
     "A": ["216.150.1.1"],
-    "TXT": "peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"
+    "TXT": ["peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"]
```

its just a single TXT record and for readability
```suggestion
    "TXT": "peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"
```